solidity/script/avs/DeployNetwork.s.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+import "forge-std/Script.sol";
+
+import {ProxyAdmin} from "../../contracts/upgrade/ProxyAdmin.sol";
+import {TransparentUpgradeableProxy} from "../../contracts/upgrade/TransparentUpgradeableProxy.sol";
+import {TimelockControllerUpgradeable} from "@openzeppelin/contracts-upgradeable/governance/TimelockControllerUpgradeable.sol";
+
+contract TimelockControllerInitializer is TimelockControllerUpgradeable {
+    function initialize(
+        uint256 minDelay,
+        address[] memory proposers,
+        address[] memory executors,
+        address admin
+    ) public initializer {
+        __TimelockController_init(minDelay, proposers, executors, admin);
+    }
+}
+
+contract DeployNetwork is Script {
+    uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+
+    address networkRegistry = 0xC773b1011461e7314CF05f97d95aa8e92C1Fd8aA;
+
+    address proxyAdmin = 0x75EE15Ee1B4A75Fa3e2fDF5DF3253c25599cc659;
```

Comment:
Very nitty optional: Create a script that pulls these from config and add as ENV var here